manageiq-operator/api/v1alpha1/helpers/miq-components/httpd.go

```diff
@@ -17,7 +19,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	intstr "k8s.io/apimachinery/pkg/util/intstr"
-	"net/http"
```

Comment:
You just can't leave it alone.  It has to be sorted by proper scope/context.  🤣

Reply:
It wasn't me haha